dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java

```diff
@@ -249,6 +249,8 @@ private Constants() {
 
     public static final String WORKER_RESERVED_MEMORY = "worker.reserved.memory";
 
+    public static final String WORKER_WEIGHT = "worker.weight";
```

Comment:
Hi,

This variable is not used in this project, please delete.
If you have any question or suggestion, please put forward~